portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -29,20 +31,29 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that deletes all comments from Datastore */
 @WebServlet("/delete-comments")
 public class DeleteServlet extends HttpServlet {
 
-    @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-
-    }
-
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-
+        System.out.println("in doPost");
+        deleteDatastoreComments(request);
+        return;
     }
 
-
+    private void deleteDatastoreComments(HttpServletRequest request) {
+        Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
If you are deleting all of the comments, it isn't clear why you would need to sort them.